integration-tests/checkstyle.sh

```diff
@@ -33,4 +32,4 @@ fi
   "${patch_error_prone_flags}" \
   "${validation_error_prone_flags}" \
   "${validation_build_flags}" \
-  $@
+  ${@}
```

Comment:
That works currently because we have only one additional flag to pass along, but is not what is intended: the idea is that remaining arguments are passed along individually.

Reply:
Clear, lets merge!